### PR TITLE
Add IAQ sensor and power monitoring sensor

### PR DESCRIPTION
**Pull request by arthur-arwin**
#### Summary

Add IAQ sensor and power monitoring sensor

#### Changes

- ...
- ...

#### Checklist for Reviewers

- [ ] Title and description should be descriptive (Not just a serial number for example).
- [ ] `vendorProfileID` should not be `vendorID`.
- [ ] All devices should be listed in the vendor's `index.yaml` file.
- [ ] At least 1 image per device and should be transparent.

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/arwin-technology/lrs2m001-4xxx.png
Comment (LDannijs) [inline vendor/arwin-technology/lrs2m001-4xxx.png:1]: Hello,

Could you please make sure the image has a transparent background?

File: vendor/arwin-technology/lrs10701.jpg
Comment (LDannijs) [inline vendor/arwin-technology/lrs10701.jpg:1]: This one transparent background as well please.

Comment (arthur-arwin) [reply vendor/arwin-technology/lrs10701.jpg:1]: I've change the background of both the pictures to transparent.

Comment (LDannijs) [reply vendor/arwin-technology/lrs10701.jpg:1]: Seems it's giving an error in the validation. Please make sure the image is correctly formatted and that the name of the image file matches what's inside the device file

Comment (arthur-arwin) [reply vendor/arwin-technology/lrs10701.jpg:1]: Just fixed it

Comment (LDannijs) [reply vendor/arwin-technology/lrs10701.jpg:1]: Thanks! My colleague @Jaime-Trinidad will have to approve it further.

**Review by Jaime-Trinidad — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
Hi @arthur-arwin thanks for adding the devices, please update this branch so we can merge it.

**Comment by arthur-arwin** [comment]
> Hi @arthur-arwin thanks for adding the devices, please update this branch so we can merge it.

Thx! I've updated the branch.

**Pull request merged**